Fix a few grammar problems (#125)
cuelang/cuelang.org: content/en/docs/usecases/configuration.md

CUE basic operation merges configurations in a way that the outcome is
always the same regardless of the order in which it is carried out
(it is associative, commutative and idempotent).

> This property enables many of its favorable properties, as discussed below.

Reviewer: > This property enables many of its favorable properties, as discussed below.

I actually think this sentence was intended as something like:

> This property is the foundation for many other favorable properties, as discussed below.

Author: That seems like a good suggestion, but I am still at the beginning of learning Cue, so I will defer to the project owners to make a call on it.

Reviewer: If you're happy making this change as part of this PR please do. Otherwise I'm happy to make it in a follow up. Thanks again

Author: Done.

> This property enables many of its favorable properties, as discussed below.

Author: ```suggestion
This property is the foundation for many other favorable properties, as discussed below.
```
